### DISCUSSION TITLE
갑자기 CUDA Out Of Memory Error가 뜰 때

### DISCUSSION THREAD

**Discussion by intrandom5** [boostcampaitech4nlp2/level1_semantictextsimilarity_nlp-level1-nlp-13#15] [Ideas]
모델이 원래 학습이 되야 하는데 갑자기 CUDA OOM 에러가 발생한다?
>
    nvidia-smi
명령어로 현재 cuda 메모리를 쓰고 있는지 확인합니다.
![image](https://user-images.githubusercontent.com/68216991/198301045-e4573d7f-2c6a-4214-a7e1-d0c35490b8ea.png)
오우 쉣 뭔가가 3000MB 정도를 쓰고 있습니다.
>
    ps -ef | grep python
명령어로 실행되어 있는 python 프로세스를 확인합니다.
![image](https://user-images.githubusercontent.com/68216991/198301264-766a3f9a-5692-4012-b857-82d257667c21.png)
왼쪽에서 2번째에 있는 숫자들이 실행중인 python 프로세스들의 PID 번호입니다.
kill 명령어를 통해 실행중인 프로세스를 종료할 수 있습니다.
>
    kill -9 52 70 3108 3110 3125 8812 9341 13026 17389 17502
    nvidia-smi
![image](https://user-images.githubusercontent.com/68216991/198301482-9bf2e83e-abf5-4d2c-925e-23337d6201d2.png)
클-린해진 것을 확인할 수 있습니다. 이제 신나게 다시 학습하면 되겠죠?

**Comment by papari1123** [comment]
오우 꿀팁 감사합니다~
